src/ui/layout/RecentObjectsList.vue

```diff
@@ -193,18 +193,10 @@ export default {
                 && this.openmct.composition.supportsComposition(domainObject);
         },
         /**
-         * Clears the Recent Objects list from localStorage.
+         * Clears the Recent Objects list.
          */
         clearRecentObjects() {
-            localStorage.removeItem(LOCAL_STORAGE_KEY__RECENT_OBJECTS);
-        },
-        /**
-         * Clears the Recent Objects list from localStorage and updates the recents list.
-         * This method is called when the user clicks the "Clear recent objects list" button.
-         */
-        handleClearAndUpdateRecentObjects() {
-            this.clearRecentObjects();
-            this.getSavedRecentItems();
+            this.recents = [];
```

Comment:
Ah, sorry-- I think what I wrote in the first review was kind of confusing. 

We want to clear RecentObjects both in `localStorage` _and_ locally here. So let's just add the line back in:

```js
localStorage.removeItem(LOCAL_STORAGE_KEY__RECENT_OBJECTS);
this.recents = [];
```

Reply:
Ohh. Sorry, I thought we had changed strategy in this behavior. 😅
I put it back. 👍